### DISCUSSION TITLE
Is SMS verification not working for Ukraine?

### DISCUSSION THREAD

**Discussion by dmytroux** [vercel/community#3003] [Help] [answered]
### Summary

Hey! 

Would like to start using Vercel, but can't Sign up, through email and Git, since in both ways need to confirm registration by phone, I don't receive any messages on my phone.
I don't see any counter or some info regarding how many times should I wait, can't resend the message as well.

Looking forward to start using the product.

### Steps to Reproduce

Sign up, and Sign In buttons on the website.
https://vercel.com/login

**Comment by swarnava** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
